### ISSUE TITLE
Move sidebar to right side of research note and wiki pages

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#3091]
This has been marked as a good candidate for becoming a [first-timers-only issue](http://www.firsttimersonly.com) like [these](https://publiclab.github.io/community-toolbox/#r=all), meaning that it's simple, self-contained, and with some extra formatting, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider [reformatting or reposting it as a first-timers-only issue](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors), and then ping @publiclab/reviewers to get it labelled. Or, if this is not your first time, try to solve it yourself!

****

We had a great UI analysis by Siyuan a while back, suggesting the sidebar should be on the right side:

> we could look at some of what Siyuan suggested re: moving sidebar to the right side (see the last comment): https://publiclab.org/notes/siyuan/06-12-2017/a-questions-page-to-encourage-discussions-through-better-ux

The templates are here, starting with notes:

https://github.com/publiclab/plots2/blob/5c94fe54e1a3fed19fb0de2c910d5478f03d2391/app/views/notes/show.html.erb#L3

Wiki pages:

https://github.com/publiclab/plots2/blob/5c94fe54e1a3fed19fb0de2c910d5478f03d2391/app/views/wiki/show.html.erb#L28-L40

Question page: 

https://github.com/publiclab/plots2/blob/5c94fe54e1a3fed19fb0de2c910d5478f03d2391/app/views/questions/show.html.erb#L7

Watch out for the proper nesting of the Bootstrap `col-md-md` class divs! https://getbootstrap.com/docs/3.3/

Once we complete this, we can replace the "related content" in the "related" sidebar with a simpler tag listing, like this:

https://user-images.githubusercontent.com/24359/37424326-8ff2290c-2796-11e8-9620-54c6d5667d62.png

So let's break that out as its own issue once this is done. 

Any help appreciated!

**Comment by derek-allen** [comment]
If it's alright, I'll go ahead and do this!

**Comment by jywarren** [comment]
👍🏼👍🏼😄⚡
